server/impl/tools.go

```diff
@@ -0,0 +1,7 @@
+// +build tools
+
+package impl
+
+import (
+	_ "golang.org/x/tools/cmd/stringer"
```

Comment:
This declares a dependency on the stringer CLI tool, which is needed for the `go:generate` lines elsewhere.